lib/stripped-build-plugins.js

```diff
@@ -36,7 +37,26 @@ module.exports = function(environment) {
           'warn',
           'debugSeal'
         ]
-      })
+      }),
+      // comment out when running non-baseline production benchmarks
+      // WARNING do not ever commit the commented out version!
+      stripHeimdall
```

Comment:
would prefer an environment variable here. inevitably this comment to not commit it will be skipped or not seen in a code review.